src/playersElo.js

```diff
@@ -1,15 +1,21 @@
 import { getPersonalScoreboard } from "./scoreboard";
 
-let url = "https://haxrecordings.s3.amazonaws.com/scoreboard.json";
+const playersElo = async function () {
+  const url = process.env.BASE_API_URL + "/players";
+  let elos = {};
+  await fetch(url)
+    .then(data => data.json())
+    .then(players => {
+      players.forEach(player => {
+        elos[player.name] = player.elo;
+      });
+    });
 
-let playersElo = async function () {
-  const result = await fetch(url)
-
-  return result.json();
+  return elos;
 };
 
-const playersEloInJsonFormat = async function () {
-  const previousElos = await playersElo();
+const playersEloInJsonFormat = function () {
+  const previousElos = playersElo();
```

Comment:
Oh I see, you did the `double nelson` here in order to avoid to put the **await** explicitly here. I didn't know that could work